### PR TITLE
Add keynote speaker post

### PR DESCRIPTION
**Pull request by janz93**

### CODE REVIEW

File: _posts/2019-05-01-terrence-lee.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Terence Lee"
```
Comment (skrings) [inline _posts/2019-05-01-terrence-lee.md]: Please use a title like "Keynote announcement" or something similar.

File: _posts/2019-05-01-terrence-lee.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+---
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
+
+You can find Terence on Twitter as [@hone02](https://twitter.com/hone02) and on GitHub as [hone](https://github.com/hone).
```
Comment (skrings) [inline _posts/2019-05-01-terrence-lee.md]: Please add a link to our ticket sales https://ti.to/ruby-unconf/2019

"Buy a ticket to attend RubyUnconf, become a potential speaker and see Terence live on stage"

File: _posts/2019-05-01-terrence-lee.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Terence Lee"
+---
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
```
Comment (skrings) [inline _posts/2019-05-01-terrence-lee.md]: Add a first paragraph where you announce Terence as the keynote speaker that will be used as excerpt. Here is an idea:

"We are thrilled to announce Terence Lee as the keynote Speaker of RubyUnconf 2019! Terence will held the keynote talk about Ruby on Saturday the 25th of May. **Add more text here**. Thank you Terence."

**Comment by skrings** [comment]
Hey @janz93 thanks for starting this! You explained very good who Terence Lee is but missed the chance to announce him as a keynote speaker. For me the blog post feels a bit half done, I added some comments with ideas how to improve the post.

File: _posts/2019-05-01-terrence-lee.md (hunk 1 above)
Comment (janz93) [reply _posts/2019-05-01-terrence-lee.md]: how about that?

File: _posts/2019-05-01-terrence-lee.md (hunk 3 above)
Comment (janz93) [reply _posts/2019-05-01-terrence-lee.md]: Yes that paragraph is great.

**Comment by janz93** [comment]
Thanks for the feedback. I updated the blog post. It is sadly a new file as I changed the name as well. @skrings could you take a look at it again?

File: _posts/2019-05-02-terrence-lee-keynote.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Keynote speaker Terence Lee"
+---
+We are thrilled to announce Terence Lee as the keynote Speaker of Ruby Unconf 2019! Terence will held his keynote about Ruby on Saturday the 25th of May.
```
Comment (skrings) [inline _posts/2019-05-02-terrence-lee-keynote.md]: `will held his keynote` -> `will give his keynote talk`

I know I am correcting my own suggestion here 😄

File: _posts/2019-05-02-terrence-lee-keynote.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Keynote speaker Terence Lee"
```
Comment (skrings) [inline _posts/2019-05-02-terrence-lee-keynote.md:3]: The title works really well!

File: _posts/2019-05-02-terrence-lee-keynote.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+---
+We are thrilled to announce Terence Lee as the keynote Speaker of Ruby Unconf 2019! Terence will held his keynote about Ruby on Saturday the 25th of May.
+
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
+
+You can find Terence on Twitter as [@hone02](https://twitter.com/hone02) and on GitHub as [hone](https://github.com/hone).
+
+Thank you Terence for all the inspiring work for our great community!
+
+[Buy a ticket](https://ti.to/ruby-unconf/2019) to attend Ruby Unconf, become a potential speaker and see Terence live on stage
```
Comment (skrings) [inline _posts/2019-05-02-terrence-lee-keynote.md]: Missing punctuation at end of line. 😱

File: _posts/2019-05-02-terrence-lee-keynote.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Keynote speaker Terence Lee"
+---
+We are thrilled to announce Terence Lee as the keynote Speaker of Ruby Unconf 2019! Terence will held his keynote about Ruby on Saturday the 25th of May.
+
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
+
```
Comment (skrings) [inline _posts/2019-05-02-terrence-lee-keynote.md:9]: Do we have a nice picture of Terence we could add here? 📸

Comment (janz93) [reply _posts/2019-05-02-terrence-lee-keynote.md:9]: Ah thanks for the reminder

**Review by skrings — APPROVED** [review]
Awesome @janz93 💯

**Pull request merged**